.github/workflows/npm-audit-fix.yml

```diff
@@ -0,0 +1,40 @@
+name: Run npm audit fix
+on: 
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 0" # Every Sunday
+
+jobs:
+  npm-audit-fix:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup node
+      uses: actions/setup-node@v1
+      with:
+        node-version: '10.x'
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Run npm audit fix
+      env:
+        GITHUB_TOKEN: ${{ secrets.ZAPBOT_TOKEN }}
+      run: |
+        # Setup git details
+        export GITHUB_USER=zapbot
+        git config --global user.email "12745184+zapbot@users.noreply.github.com"
+        git config --global user.name $GITHUB_USER
+        # Run the npm commands
+        npm update
+        npm audit fix
+        git checkout -b npm-audit-fix
+        # Update the index to be sure git is aware of changes
+        git update-index -q --refresh
+        if ! git diff-index --quiet HEAD --; then
+          # Generate a PR
+          git remote set-url origin https://$GITHUB_USER:$GITHUB_TOKEN@github.com/$GITHUB_USER/zaproxy-website.git
+          git add package*
+          git commit -m "npm audit fix" --signoff
+          git push --set-upstream origin npm-audit-fix --force
+          hub pull-request -b zaproxy:master --no-edit
```

Comment:
Is the -b really needed, the man page says this is the default

Reply:
Not sure tbh - copied from https://github.com/zaproxy/zap-admin/blob/master/.github/workflows/alert-docs.yml#L39 :)

Reply:
Alright, well it isnt hurting anything i guess.

Reply:
This is needed to raise the PR in `zaproxy` (otherwise it raises it in `zapbot`), there's no `upstream` remote defined.